### PR TITLE
Use embedded node in ruling and QA

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -142,25 +143,31 @@ ws_scan_task:
     ws_artifacts:
       path: 'whitesource/**/*'
 
-plugin_qa_task:
+plugin_qa_with_node_task:
   <<: *PLUGIN_QA_BODY
   eks_container:
     dockerfile: .cirrus/nodejs.jdk17.Dockerfile
-    docker_arguments:
```
Comment (ilia-kebets-sonarsource) [inline .cirrus.yml:149]: so what is the node version on which we test?

File: .cirrus.yml [hunk 2]
```diff
@@ -189,7 +196,7 @@ ruling_task:
   <<: *ONLY_SONARSOURCE_QA
   eks_container:
     <<: *CONTAINER_DEFINITION
-    dockerfile: .cirrus/nodejs.jdk17.Dockerfile
+    image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-latest
```
[the remark this answers is not in the document]
Comment (saberduck) [reply .cirrus.yml:199]: yes

File: .cirrus.yml (hunk 1 above)
Comment (saberduck) [reply .cirrus.yml:149]: it's 14, default value is inside the dockerfile

**Review by ilia-kebets-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**